src/drivers/distance_sensor/lightware_laser_i2c/lightware_laser_i2c.cpp

```diff
@@ -570,10 +570,16 @@ extern "C" __EXPORT int lightware_laser_i2c_main(int argc, char *argv[])
 	int ch;
 	using ThisDriver = LightwareLaser;
 	BusCLIArguments cli{true, false};
-	cli.rotation = (Rotation)distance_sensor_s::ROTATION_DOWNWARD_FACING;
+	// cli.rotation = (Rotation)distance_sensor_s::ROTATION_DOWNWARD_FACING;
+	// cli.rotation = (Rotation)distance_sensor_s::ROTATION_YAW_45;
+	// cli.rotation = (Rotation)distance_sensor_s::ROTATION_RIGHT_FACING;
+	// cli.rotation = (Rotation)distance_sensor_s::ROTATION_LEFT_FACING;
```

Comment:
```suggestion
```
We don't need this, if we want to change the rotation we should add a parameter for it.